BraveShared/tr.lproj/BraveShared.strings

```diff
@@ -0,0 +1,1353 @@
+"HideLinkPreviewsActionTitle" = "Bağlantı Önizlemelerini Gizle";
+
+/* Hides the rewards icon */
+"HideRewardsIcon" = "Brave Ödülleri Simgesini Gizle";
+
+/* Hide the rewards icon explination. */
+"HideRewardsIconSubtitle" = "Brave Ödülleri etkinleştirilmediğinde Brave Ödülleri simgesini gizler";
+
+/* Title for history menu item */
+"HistortMenuItem" = "Geçmiş";
+
+/* Title for history screen */
+"HistoryScreenTitle" = "Geçmiş";
+
+/* No comment provided by engineer. */
+"Home" = "Ana Sayfa";
+
+/* No comment provided by engineer. */
+"HTTPSEverywhere" = "Bağlantıları HTTPS'ye yükselt";
+
+/* No comment provided by engineer. */
+"HTTPSEverywhereDescription" = "Bazı siteleri düz HTTP yerine güvenli bir HTTPS bağlantısı kullanarak açar.";
+
+/* individual blocking statistic title */
+"HTTPSUpgrades" = "HTTPS Yükseltmeleri";
+
+/* title for per-site shield toggles */
+"IndividualControls" = "Bireysel Denetimler";
+
+/* Sync Alert */
+"InvalidSyncCodeDescription" = "Geçersiz Eşitleme Kodu";
+
+/* History tableview section header */
+"LastMonth" = "Geçen ay";
+
+/* History tableview section header */
+"LastWeek" = "Geçen hafta";
+
+/* DuckDuckGo promotion label. */
+"LearnAboutPrivateSearchrwithDuckDuckGo" = "Gizli arama hakkında bilgi edinin \nDuckDuckGo ile birlikte";
+
+/* No comment provided by engineer. */
+"learnMore" = "Daha fazla bilgi edinin";
+
+/* Footer label describing when the current login was last modified with the timestamp as the parameter. */
+"LoginDetailLastModifiedCellFormatTitle" = "Son değiştirilen % @";
+
+/* Label displayed above the password row in Login Detail View. */
+"LoginDetailPasswordCellTitle" = "parola";
+
+/* Label displayed above the username row in Login Detail View. */
+"LoginDetailUsernameCellTitle" = "kullanıcı adı";
+
+/* Label displayed above the website row in Login Detail View. */
+"LoginDetailWebsiteCellTitle" = "web sitesi";
+
+/* Label for the button used to deselect all logins. */
+"LoginListDeselectAllButtonTitle" = "Tüm Seçimleri Kaldır";
+
+/* Label displayed when no logins are found after searching. */
+"LoginListNoLoginTitle" = "Oturum açma bilgisi bulunamadı";
+
+/* Title for Logins List View screen. */
+"LoginListScreenTitle" = "Oturum açma bilgileri";
+
+/* Label for the button used to select all logins. */
+"LoginListSelectAllButtonTitle" = "Tümünü seç";
+
+/* Setting to allow media to play automatically */
+"MediaAutoPlays" = "Arka planda video otomatik oynatmayı etkinleştir";
+
+/* Copy password text selection menu item */
+"MenuItemCopyTitle" = "Kopyala";
+
+/* Hide password text selection menu item */
+"MenuItemHidePasswordTitle" = "Gizle";
+
+/* Open and Fill website text selection menu item */
+"MenuItemOpenAndFillTitle" = "Aç ve Doldur";
+
+/* Reveal password text selection menu item */
+"MenuItemRevealPasswordTitle" = "Göster";
+
+/* Abbreviation for 'Month', use full word' Month' if this word can't be shortened in your language */
+"monthAbbreviation" = "mo.";
```

Comment:
No need shortening in Turkish and mo. doesnt mean anything. It can be 

"monthAbbreviation" = "ay";